docker-integrations/tendermint/Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Build Stage 1
+# This build created a staging docker image
+#
+FROM node:lts AS appbuild
```

Comment:
@luis-rf I prefer we use a digest for the tag in order to be 100% reproducible. Find the node version needed for the "build" stage and use that.

Reply:
@harrytou that has the disadvantage that we don't get the latest lts security fixes.

Reply:
leaving it as LTS will allow us to cross compile for Arm and AMD64